README.md

```diff
@@ -22,30 +23,26 @@ PgJDBC regression tests are run against all PostgreSQL versions since 9.1, inclu
 Most people do not need to compile PgJDBC. You can download the precompiled driver (jar) from the [PostgreSQL JDBC site](https://jdbc.postgresql.org/download.html) or using your chosen dependency management tool:
 
 ### Maven Central
-You can search on The Central Repository with GroupId and ArtifactId [![Maven Search](https://img.shields.io/badge/org.postgresql-postgresql-yellow.svg)][mvn-search] for:
+You can search on The Central Repository with GroupId and ArtifactId [![Maven Search](https://img.shields.io/badge/org.postgresql-postgresql-yellow.svg)][mvn-search]:
 
-[![Java 8](https://img.shields.io/badge/Java_8-42.3.1-blue.svg)][mvn-jre8]
-```xml
-<dependency>
-    <groupId>org.postgresql</groupId>
-    <artifactId>postgresql</artifactId>
-    <version>42.3.1</version>
-</dependency>
-```
-
-[mvn-search]: https://search.maven.org/search?q=g:org.postgresql%20AND%20a:postgresql "Search on Maven Central"
-[mvn-jre8]: https://search.maven.org/artifact/org.postgresql/postgresql/42.3.1/jar
+[![Maven Central](https://img.shields.io/badge/maven--central-postgresql-informational?style=for-the-badge&logo=apache-maven&logoColor=red)](https://maven-badges.herokuapp.com/maven-central/org.postgresql/postgresql)
```

Comment:
Do we really need two buttons there? (see https://github.com/jorsol/pgjdbc/tree/readme-version-update#get-the-driver)

I think a single one is enough. For instance, this should be just ok: [![Maven Central](https://maven-badges.herokuapp.com/maven-central/org.postgresql/postgresql/badge.svg)](https://search.maven.org/artifact/org.postgresql/postgresql) (it displays the recent version, and it directs to the list of all published versions)

Reply:
Well, the idea is that the first button links to the list of all published versions, and the second links to the latest published version.

So if you like only one button, I could link to the the latest published version which IMO is what we really want.

Reply:
Well, the most important information is "the exact last version". So if we could just display the badge with the latest version, then it would cover 99.42% of all the needs.

The two buttons do not fit on the screen, and the styles of the butts are completely different, so I suggest we just have a single one with the latest version.